business_requirement_renew/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+==========================
+Business Requirement Renew
+==========================
+
+This module adds an renew button, and if you click on it you will get the
```

Comment:
Add the introduction of the other modules.
Please prepare screenshots with a simple procedure in Usage section